Doc/library/idle.rst

```diff
@@ -660,29 +660,61 @@ functions to be used from IDLE's Python shell.
 Command line usage
 ^^^^^^^^^^^^^^^^^^
 
-.. code-block:: none
+.. program:: idle
 
-   idle.py [-c command] [-d] [-e] [-h] [-i] [-r file] [-s] [-t title] [-] [arg] ...
+IDLE can be invoked from the command line with various options. The general syntax is:
 
-   -c command  run command in the shell window
-   -d          enable debugger and open shell window
-   -e          open editor window
-   -h          print help message with legal combinations and exit
-   -i          open shell window
-   -r file     run file in shell window
-   -s          run $IDLESTARTUP or $PYTHONSTARTUP first, in shell window
-   -t title    set title of shell window
-   -           run stdin in shell (- must be last option before args)
+.. code-block:: bash
 
-If there are arguments:
+   idle [options] [file ...]
 
-* If ``-``, ``-c``, or ``r`` is used, all arguments are placed in
-  ``sys.argv[1:...]`` and ``sys.argv[0]`` is set to ``''``, ``'-c'``,
-  or ``'-r'``.  No editor window is opened, even if that is the default
-  set in the Options dialog.
+The following options are available:
+
+.. option:: -c <command>
+
+   Run the specified Python command in the shell window. For example:
+
+   .. code-block:: bash
+
+      idle -c "print('Hello, World!')"
```

Comment:
```suggestion
   Run the specified Python command in the shell window. For example, pass ``-c "print('Hello, World!')"``.
On Windows, the outer quotes must be double quotes as shown.  
```
Can we break the line?  If so, skip indent on next line as above?  Some other lines below are over 80 chars.

Reply:
There was no blank line between the two + lines in the suggestion.  I removed it now and am retesting.